portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,40 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import com.google.gson.Gson;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.swing.ListModel;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Neal!");
+    List<String> names = new ArrayList<String>();
+    names.add("Hello Neal!");
+    names.add("Hello Bob!");
+    names.add("Hello Joe!");
+    String json = convertToJsonUsingGson(names);
+    response.setContentType("application/json;");  
+    response.getWriter().println(json);
+  }
+
+ /**
+   * Converts the input of an arraylist of names into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
```

Comment:
Nit: this could have nicer line breaking.

Reply:
When you say nicer does that mean less characters per line?

Reply:
Yes. But It's fine.